### PR TITLE
Fix H1 Issues in Docs

### PR DESCRIPTION
**Pull request by benarent**
We are only allowed to have one H1 per page for SEO reasons. This PR cleans up the docs to either remove redundant H1s or bump them to H2s.

### CODE REVIEW

**Review by avatus — APPROVED** [review]

**Review by alexfornuto — APPROVED** [review]
Thanks! I made a few additional changes, including one title change to avoid redundancy. Since I contributed, @ptgott should also give an independent review.

File: docs/pages/application-access/getting-started.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Getting started with Teleport Application Access.
 videoBanner: 5Uwhp3IQMHY
 ---
 
-# Getting Started
+## Getting Started with Teleport Application Access
```
Comment (alexfornuto) [inline docs/pages/application-access/getting-started.mdx:8]:
```suggestion
```

File: docs/pages/database-access/rbac.mdx [hunk 2]
```diff
@@ -1,10 +1,8 @@
 ---
-title: Database Access Access Controls
+title: Database Access Role-Based Controls
```
Comment (alexfornuto) [inline docs/pages/database-access/rbac.mdx:2]: This is the one change I'd like an independent review of.

**Review by alexfornuto — APPROVED** [review]

File: docs/pages/application-access/getting-started.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/application-access/getting-started.mdx:8]: I missed one in my first review.

**Comment by alexfornuto** [comment]
@benarent Any reason not to backport beyond 12?

File: docs/pages/database-access/rbac.mdx (hunk 2 above)
Comment (ptgott) [reply docs/pages/database-access/rbac.mdx:2]: This looks a bit awkward to me. Would "Role-Based Access Controls for Databases" make sense here?

**Comment by benarent** [comment]
> @benarent Any reason not to backport beyond 12?

Let's start with 12 and 11, then see what the SEO consultants tell us.

File: docs/pages/database-access/rbac.mdx (hunk 2 above)
Comment (benarent) [reply docs/pages/database-access/rbac.mdx:2]: How about 

Database Role-Based Access Controls ?

Comment (alexfornuto) [reply docs/pages/database-access/rbac.mdx:2]:
```suggestion
title: Database Role-Based Access Controls
```

**Pull request merged**